### ISSUE TITLE
Figure out a way to display a title for individual charts

### ISSUE THREAD

**Issue by jimmybutton** [simonw/django-sql-dashboard#153]

**Comment by simonw** [comment]
Yeah I've been using that same trick - having queries which do `select '## Users per day' as markdown` - and it's not exactly elegant.

I like your suggestion of having an optional title.
